scripts/doctor

```diff
@@ -33,14 +33,14 @@ count_in_path() {
 }
 
 
-count_in_path "rbenv"
+count_in_path "ruby"
```

Comment:
I think that the problem with a raw Ruby check is that macs come with an unusable version installed. I'm not sure of a solid way to check specific versions, though.

Reply:
Modified it to check that version minimum version 3.1.0 is installed.